runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/PipelineTranslation.java

```diff
@@ -155,27 +155,21 @@ private static void addRehydratedTransform(
     // Only ParDo really separates main from side inputs
```

Comment:
<!--new_thread; commit:bd1ae5fc000332b3113096a9d6b58194924e60cc; resolved:0-->
Drop ``// Only ParDo ...` since this is applying to WriteFilesTranslation.